src/openvino.cc

```diff
@@ -737,14 +737,14 @@ ModelInstanceState::ModelInstanceState(
   }
 
   if (model_state_->NetworkNotRead()) {
+    THROW_IF_BACKEND_INSTANCE_ERROR(model_state_->ParseParameters(device_));
     THROW_IF_BACKEND_INSTANCE_ERROR(
         model_state_->ReadNetwork(ArtifactFilename(), &model_path_));
     THROW_IF_BACKEND_INSTANCE_ERROR(model_state_->ParseParameters());
     THROW_IF_BACKEND_INSTANCE_ERROR(model_state_->ValidateConfigureNetwork());
   }
 
   if (model_state_->NetworkNotLoaded(device_)) {
-    THROW_IF_BACKEND_INSTANCE_ERROR(model_state_->ParseParameters(device_));
```

Comment:
How about moving `LoadCpuExtensions()` from `ModelState::ParseParameters(const std::string& device)` to `ModelState::ParseParameters()`? I think even this should do the trick and solve the bug.

Can you try if this fix works and update the PR?

Reply:
ok, I will test, thank you very much

Reply:
I just looked at the code, move LoadCpuExtensions to ModelState: : ParseParameters () should not be good, because the ModelState: : ParseParameters execute before model_state_ - > ReadNetwork, if not load extension before ReadNetwork  an error will be triggered, Unless ReadNetwork is executed after LoadCpuExtensions

Reply:
Sorry for not being clear enough. Please move the LoadCpuExtenstions to ModelState: : ParseParameters ()  and move  ModelState: : ParseParameters () call right before ReadNetwork(). I am asking this because in future we might add support for other devices in openvino backend and having a separate device specific parser for network loading will be more intuitive.

Reply:
Can you move the LoadCpuExtensions to ModelState::ParseParameters() and then move  ModelState::ParseParameters() call to right before ReadNetwork() call. The reason I am asking this is because we will be adding support for other devices in OV backend some time later and having a clear device agnostic network read and device specific network load will be helpful.

Reply:
ok, I will change my code.